src/ocnicepost.fd/ocnicepost.F90

```diff
@@ -0,0 +1,577 @@
+program ocnicepost
+
+  use utils_mod, only : debug, logunit, getfield, packarrays, remap, dumpnc
+  use outputvars
+  use netcdf
+
+  implicit none
+
+  character(len= 20) :: fnml, ftype
+  character(len=120) :: wgtsdir
+  character(len=120) :: input_file, wgtsfile, output_file
+
+  ! source grid, tripole 1/4 deg, 40 vertical levels
+  integer, parameter :: nxt = 1440, nyt = 1080, nlevs = 40
+
+  ! destination grids
+  integer, parameter :: ndest = 3
+  integer, parameter, dimension(ndest) :: nxrs = (/1440, 720, 360/)
+  integer, parameter, dimension(ndest) :: nyrs = (/ 721, 361, 181/)
+  character(len=4), dimension(ndest)   :: dstgrds = (/'0p25', '0p5 ', '1p0 '/)
```

Comment:
Hard coding of variables is not supported. The hard coded attributes in this instance should be read into the program via either a FORTRAN namelist or a CSV file.